_posts/2021-12-16-jaas-realm.adoc

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: 'Using JAAS realm in Elytron'
+date: 2021-12-16
+tags: elytron wildfly JAAS security realm
```

Comment:
With Ashpan's updates to the tags (see https://github.com/wildfly-security/wildfly-elytron/pull/1634), we can remove the elytron and wildfly tags here and then update the others to "jaas" and "security-realm".

Reply:
Fixed.

Reply:
changed to login-module and login-context to be consistent, sorry